### PR TITLE
Nrf52 dfu touch1200, remove bootloader target

### PR DESCRIPTION
**Pull request by hathach**
This PR does following things with nrf52

- remove `bootloader`, `dfu-boot(loader)` target, along with bootloader hex/zip file in the boards folder. User should clone the https://github.com/adafruit/Adafruit_nRF52_Bootloader and flash there. It is possible to flash or upgrade (dfu-flash) to specific version without compiling the bootloader. Also update readme.md for #1069 
- `adafruit-nrfutil` is used instead of nrfutil, readme.md is also updated
- only new feature is touch 1200, when the Circuitpython CDC is disconnected @1200bps, the board will go into CDC-only DFU. Along with the new adafruit-nrfutl 0.5.3post9. Upgrading firmware does not require us to manually put the board into DFU. Just simply `make BOARD=pca10056 dfu-flash` should do.

Note: on macos, the SERIAL name changes e.g /dev/tty.usbmodem26 (CP mode) to /dev/tty.usbmodem1124  (dfu mode), I don't know how to go around this. on Linux the ttyACM appears to be the same after reboot.

### CODE REVIEW

File: ports/nrf/README.md [hunk 1]
```diff
@@ -98,11 +99,9 @@ note: On Linux it might be required to link SEGGER's `libjlinkarm.so` inside nrf
...
 ## DFU Targets
 
-    sudo apt-get install build-essential libffi-dev pkg-config gcc-arm-none-eabi git python python-pip
-    git clone https://github.com/adafruit/Adafruit_nRF52_Arduino.git
-    cd Adafruit_nRF52_Arduino/tools/nrfutil-0.5.2/
-    sudo pip install -r requirements.txt
-    sudo python setup.py install
+run follow command to install [adafruit-nrfutil](https://github.com/adafruit/Adafruit_nRF52_nrfutil) from PyPi
+
+    $ pip3 install adafruit-nrfutil --user
```
Comment (dhalbert) [inline ports/nrf/README.md]: Just change this to
```
$ pip3 install --user adafruit-nrfutil
```
which is the standard order of options followed by args (it doesn't matter to the command, but that's the usual order).

File: ports/nrf/boards/feather52832/README.md [hunk 2]
```diff
@@ -21,24 +21,15 @@ $ cd ports/nrf
...
+## Installing `adafruit-nrfutil`
 
 The Adafruit Bluefruit nRF52 Feather ships with a serial and OTA BLE bootloader
 that can be used to flash firmware images over a simple serial connection,
 using the on-board USB serial converter.
 
-If you haven't installed this command-line tool yet, go to the `/libs/nrfutil`
-folder (where nrfutil 0.5.2 is installed as a sub-module) and run the following
-commands:
+run follow command to install [adafruit-nrfutil](https://github.com/adafruit/Adafruit_nRF52_nrfutil) from PyPi
```
Comment (dhalbert) [inline ports/nrf/boards/feather52832/README.md]: "run follow command" -> "Run the following command"

File: ports/nrf/boards/feather52832/README.md [hunk 3]
```diff
@@ -21,24 +21,15 @@ $ cd ports/nrf
...
 
-> If you get a 'sudo: pip: command not found' error running 'sudo pip install',
-you can install pip via 'sudo easy_install pip'
-
-```
-$ cd ../../lib/nrfutil
-$ sudo pip install -r requirements.txt
-$ sudo python setup.py install
-```
+    $ pip3 install adafruit-nrfutil --user
```
Comment (dhalbert) [inline ports/nrf/boards/feather52832/README.md]: `pip3 install --user adafruit-nrfutil`

File: ports/nrf/boards/feather52840/README.md [hunk 4]
```diff
@@ -169,8 +161,7 @@ BUTTON1 still pressed as you come out of reset).
 This will give you a **fast blinky DFU pattern** to indicate you are in DFU
 mode.
 
-At this point, you can **build and flash** a CircuitPython binary via the following
-command:
+you can **build and flash** a CircuitPython binary via the following command:
```
Comment (dhalbert) [inline ports/nrf/boards/feather52840/README.md]: "you" -> "You"

File: ports/nrf/boards/feather52840/README.md [hunk 5]
```diff
@@ -197,19 +188,19 @@ Device programmed.
 
 ### Flashing CircuitPython with MSC UF2
 
-Make `uf2` target to generate the uf2
+uf2 file is generated at the end of `all` target
```
Comment (dhalbert) [inline ports/nrf/boards/feather52840/README.md]: "The uf2 file is generated last by the `all` target"

File: ports/nrf/boards/feather52840/README.md [hunk 6]
```diff
@@ -197,19 +188,19 @@ Device programmed.
...
+../../tools/uf2/utils/uf2conv.py -f 0xADA52840 -c -o "build-feather52840-s140/firmware.uf2" "build-feather52840-s140/firmware.hex"
 Converting to uf2, output size: 392192, start address: 0x26000
 Wrote 392192 bytes to build-feather52840-s140/firmware.uf2.
 ```
 
 Simply drag and drop firmware.uf2 to the MSC, the nrf52840 will blink fast and reset after done.
 
-**Note**: you need to update `tools/uf2` for uf2conv.py to support hex file input, current circuitpython's master use older verion of uf2conv.py which only support biin file input. To update, change directory to top folder of circuitpython and run. The size of uf2 should be ~400KB, if using the old uf2conv.py the output file would be 1 MB which is not correct.
+**Note**: you need to update `tools/uf2` for uf2conv.py to support hex file input, current circuitpython's master use older verion of uf2conv.py which only support bin file input. To update, change directory to top folder of circuitpython and run. The size of uf2 should be ~400KB, if using the old uf2conv.py the output file would be 1 MB which is not correct.
```
Comment (dhalbert) [inline ports/nrf/boards/feather52840/README.md]: Is this still true? we should update uf2conv.py so this doesn't need to be done by hand.

Comment (hathach) [reply ports/nrf/boards/feather52840/README.md]: yeah, it is updated, will remove this

File: ports/nrf/boards/feather52840/README.md (hunk 4 above)
Comment (hathach) [reply ports/nrf/boards/feather52840/README.md]: You have an eagle eye :+1:

**Pull request merged**